### PR TITLE
Swap optional to the end

### PR DESCRIPTION
**Pull request by Camwyn**
🎫  https://central.tri.be/issues/123608

### CODE REVIEW

File: src/Tribe/Attendees.php [hunk 1]
```diff
@@ -803,7 +803,7 @@ public function user_can_manage_attendees( $user_id = 0, $event_id = '' ) {
 		 * @param int $user_id id of the user we're checking
 		 * @param int $event_id id of the event we're checking (matter for checks on event authorship)
```
Comment (sc0ttkclark) [inline src/Tribe/Attendees.php]: Change the param order too

**Comment by sc0ttkclark** [comment]
This filter is already out there, should we stick with the param order and fix usage in a hotfix for CT?

**Comment by Camwyn** [comment]
We can, the filter was just added last sprint.

I hate changing signatures too, but we treat `$user_id` as an optional, optionals should go to the end, and `$event_id` is _not_ optional?

**Comment by sc0ttkclark** [comment]
Need to discuss this further on Monday

**Comment by Camwyn** [comment]
Closing due to change in code strategy

**Pull request closed without merging**